testsuite/forge/src/interface/prometheus_metrics.rs

```diff
@@ -158,6 +159,12 @@ pub async fn fetch_latency_breakdown(
     let qs_batch_to_pos_query = r#"sum(rate(quorum_store_batch_to_PoS_duration_sum{role=~"validator"}[1m])) / sum(rate(quorum_store_batch_to_PoS_duration_count{role=~"validator"}[1m]))"#;
     let qs_pos_to_proposal_query = r#"sum(rate(quorum_store_pos_to_pull_sum{role=~"validator"}[1m])) / sum(rate(quorum_store_pos_to_pull_count{role=~"validator"}[1m]))"#;
 
+    // the result of mempool_to_block_creation_query is the sum of the individual limit values of qs_batch_to_pos_query and qs_pos_to_proposal_query
+    let mempool_to_block_creation_query = format!(
+        "sum({}, {})",
+        qs_batch_to_pos_query, qs_pos_to_proposal_query
+    );
```

Comment:
I think @zekun000 was mentioning that this sum was not precise, and that the query you used in the previous version of the PR is computing things better?

Reply:
> "...update all tests that use QsBatchToPos / QsPosToProposal, to use new MempoolToBlockCreation instead (just sum up their individual limit values together"

looking at our chats... I was under the impression you wanted me to do calculate the sum, but i can change to the previous query